snorkel/annotations.py

```diff
@@ -336,33 +375,88 @@ def load_matrix(self, session, split, key_group=0, **kwargs):
         return load_feature_matrix(session, split=split, key_group=key_group, **kwargs)
 
 
-def _to_annotation_generator(fns):
-    """"
-    Generic method which takes a set of functions, and returns a generator that yields
-    function.__name__, function result pairs.
-    """
-    def fn_gen(c):
-        for f in fns:
-            yield f.__name__, f(c)
-    return fn_gen
+def save_marginals(session, X, marginals, training=True):
```

Comment:
If we don't actually like the term 'marginal,' is this the time to switch to something else?